newscoop/library/Newscoop/Tools/Console/Command/SchedulerManagerCommand.php

```diff
@@ -99,7 +99,7 @@ protected function execute(Console\Input\InputInterface $input, Console\Output\O
      */
     private function cleanString($string)
     {
-        $string = str_replace(' ', '-', $string);
+        $string = str_replace(' ', '-', strtolower($string));
```

Comment:
no its not related, I just changed the names to be lowercase as it looks ugly, see a few lines above for the proper fix